js/components/organization/members.vue

```diff
@@ -72,7 +72,7 @@
              v-repeat="member:org.members">
             <a class="pointer"
                  v-on="click: member_click(member)">
-                <img class="img-circle" alt="{{ _('User Image') }}"
+                <img class="img-circle" alt="{{ _('User Image') }}" width="60" height="60"
```

Comment:
Be carefull, right now it scales to the display.
Have you tried with different display sizes ?